identity-server/src/Telemetry/Telemetry.cs

```diff
@@ -93,7 +94,8 @@ public static class TagValues
         /// <summary>
         /// Meter for experimental counters from IdentityServer
         /// </summary>
-        public static readonly Meter ExperimentalMeter = new Meter(ServiceNameExperimental, ServiceVersion);
+        [Obsolete("The ExperimentalMeter will be removed in a future version. Use the default Meter instead.")]
+        public static readonly Meter ExperimentalMeter = Meter;
```

Comment:
Similarly to `ServiceNameExperimental`, let's just get rid of this Meter (unless you think there's a use case where folks would create their own counters using the ExperimentalMeter?)